### ISSUE TITLE
on load scrollToAnchor is not enough using Firefox if page is already loaded

### ISSUE THREAD

**Issue by pulse-mind** [alvarotrigo/multiscroll.js#119]
It works fine on Chrome but not on Firefox.

Hello,

I have a case where the page is already loaded when JS is running (in jquery.multiscroll.js)
```
 $(window).on('load', function() {
                scrollToAnchor();
                });
```
If I replace by this (it works)
```
            if (document.readyState == 'complete'){
                scrollToAnchor();
            }else{
                $(window).on('load', function() {
                scrollToAnchor();
                });
            }
```

If you need I can fork and do a pull request
Otherwise tell me where I am wrong

And thank you for the job done!!

**Comment by pulse-mind** [comment]
If you need I can fork and do a pull request
Otherwise tell me where I a wrong

**Comment by alvarotrigo** [comment]
Thanks for that! 👍 
I'll take a look at it for the next release!

**Comment by pulse-mind** [comment]
Ok great, do you know when will be the next release ?

**Comment by alvarotrigo** [comment]
I don't really have a date in mind yet.

**Comment by alvarotrigo** [comment]
Fixed on the latest version multiScroll 0.2.3
